data/hai/hai reveal 6 end.txt

```diff
@@ -109,17 +117,25 @@ mission "Hai Reveal [C02] Cleanup"
 			choice
 				`	"The Hai are no longer a secret in human space, but they are concerned how you ended up here."`
 					goto revealed
+					to display
+						not "Hai Reveal [C02] Cleanup: label revealed"
 				`	"What happened to the ship that brought you here?"`
 					goto kidnappers
+					to display
+						not "Hai Reveal [C02] Cleanup: label kidnappers"
 				`	"Well, lets get you home before anything else happens."`
 					goto depart
 			label revealed
+			action
+				set "Hai Reveal [C02] Cleanup: label revealed"
 			`	"Ah, we thought we saw something about that, but our comms are spotty, so we didn't rely on it. We all actually signed up to go to Mountaintop to take part in the facade, so it seemed especially suspicious."`
 			`	He shrugs. "An abundance of caution seemed warranted; we were kidnapped, after all."`
 				goto responses
 			label kidnappers
+			action
+				set "Hai Reveal [C02] Cleanup: label kidnappers"
 			`	"We were en route to somewhere after the ship left Hai space. We don't really know what happened, but they received some kind of message that left them very concerned. They said something about hiding somewhere safe and landed here."`
-			`	At this point, he frowns in the way Hai do. "We were set down here for days and days. Our kidnappers couldn't seem to work out what they should do. Eventually, a woman landed next to us in this ship here," he explains, gesturing at the Star Barge. "She had them ferry us and all our stuff out to this ship, and then she took charge and flew off with them. That was when we sent the message. We were partly afraid it was some kind of elaborate trap when we saw that we could communicate to a Hai address directly. We never used to be able to communicate between human space and ours like that. I guess we were lucky this time."`
+			`	At this point, he frowns. "We were set down here for days and days. Our kidnappers couldn't seem to work out what they should do. Eventually, a woman landed next to us in this ship here," he explains, gesturing at the Star Barge. "She had them ferry us and all our stuff out to this ship, and then she took charge and flew off with them. That was when we sent the message. We were partly afraid it was some kind of elaborate trap when we saw that we could communicate to a Hai address directly. We never used to be able to communicate between human space and ours like that. I guess we were lucky this time."`
```

Comment:
Remember how earlier we described how Hai naturally are somewhat more alien than you might expect from encountering those familiar with humans?
This was meant to be a subtle nod to the possibility that you've seen enough of them to interpret them sometimes rather than have them adapt to you all the time.

Reply:
Perhaps, but I felt it was not very meaningful unless it was described to the player. I don't know how a Hai frowns, anyway.

Reply:
For it to be meaningful, we would have to describe some aspects of the Hai facial features, but the game is very careful not to describe the various alien species in detail. Rather, we leave it to the player's imagination.

Reply:
Which is why we describe such things like familiarity with those foreign details in abstract terms.
I'm not overly hung up on this point though, just felt I should clarify that there was intention to it.
If people don't like it it can stay removed.

Reply:
I just made a change. How about this? "At this point, he twists his face in a non-human manner you recognize as a Hai frown."